robyn/__main__.py

```diff
@@ -17,10 +17,24 @@ def create_robyn_app():
             ],
             "default": None,
         },
+        {
+            "type": "list",
+            "message": "Please select project type (Mongo/Postgres/Sqlalchemy/Prisma): ",
+            "choices": [
+                Choice("no db", name="No DB"),
+                Choice("sqlite", name="Sqlite"),
+                Choice("postgres", name="Postgres"),
+                Choice("mongo", name="MongoDB"),
+                Choice("sqlalchemy", name="SqlAlchemy"),
+                Choice("prisma", name="Prisma"),
+            ],
+            "default": None,
+        },
     ]
     result = prompt(questions=questions)
     project_dir = result[0]
     docker = result[1]
+    project_type = result[2]
```

Comment:
```suggestion
    project_dir, docker, project_type = result
```

Reply:
I tried this and it did not parse correctly so I left it as is

Reply:
@bwdq , what do you mean by this? What error were you getting?

We can return a dictionary instead of a tuple if there is an issue